src/Microsoft.AspNetCore.Server.Kestrel/WebHostBuilderKestrelExtensions.cs

```diff
@@ -28,6 +29,7 @@ public static IWebHostBuilder UseKestrel(this IWebHostBuilder hostBuilder)
             return hostBuilder.ConfigureServices(services =>
             {
                 services.AddTransient<IConfigureOptions<KestrelServerOptions>, KestrelServerOptionsSetup>();
+                services.AddTransient<IConfigureOptions<KestrelServerOptions>, KestrelServerConfigureOptions>();
```

Comment:
Doing it this way temporarily to work around https://github.com/aspnet/DependencyInjection/issues/500